DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionSwizzler.swift

```diff
@@ -6,77 +6,70 @@
 
 import Foundation
 
+/// Swizzles `URLSession` methods.
 internal class URLSessionSwizzler {
```

Comment:
/question Now that we have `URLSessionTaskSwizzler`, why do we need `URLSessionSwizzler`?
It looks like it's only there to call `interceptionWillStart` which not used by any feature.

Reply:
So we ended up removing the `URLSessionSwizzler` right?

Reply:
yes